modules/programs/vscode.nix

```
@@ -56,12 +56,29 @@ in {
         '';
       };
 
+      disableUpdateCheck = mkOption {
+        type = types.bool;
+        default = false;
+        description = ''
+          Enable to silence update popups. Updates userSettings.
+        '';
+      };
```

I think it's better to have options be in the positive direction.

```suggestion
      enableUpdateCheck = mkEnableOption "update popups";
```